package/yast2-packager.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Oct  3 16:38:36 UTC 2016 - jreidinger@suse.com
+
+- prevent double URL encoding for ISO (bsc#954813)
+- 3.1.118
```

Comment:
I need to release Leap 42.2 update, so as we agreed, bump version there and for critical maintenance updates use 3.1.117.1